functests/1_performance/cpu_management.go

```diff
@@ -415,6 +415,58 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			Expect(defaultSmpAffinitySet).To(Equal(onlineCPUsSet), fmt.Sprintf("Default SMP Affinity %s should be equal to all active CPUs %s", defaultSmpAffinitySet, onlineCPUsSet))
 		})
 	})
+
+	When("reserved CPUs specified", func() {
+		var testpod *corev1.Pod
+
+		BeforeEach(func() {
+			testpod = pods.GetTestPod()
+			testpod.Namespace = testutils.NamespaceTesting
+			testpod.Spec.NodeSelector = map[string]string{testutils.LabelHostname: workerRTNode.Name}
+
+			err := testclient.Client.Create(context.TODO(), testpod)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = pods.WaitForCondition(testpod, corev1.PodReady, corev1.ConditionTrue, 10*time.Minute)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("should run infra containers on reserved CPUs", func() {
+			podUID := strings.Replace(string(testpod.UID), "-", "_", -1)
```

Comment:
So the pod UID is guaranteed to be used in the infra-container cpuset path?

Reply:
yes

Reply:
in all the cpuset paths IIRC.